cnxarchive/utils.py

```diff
@@ -41,10 +41,11 @@ def split_ident_hash(ident_hash, split_version=False):
     if HASH_CHAR not in ident_hash:
         ident_hash = '{}@'.format(ident_hash)
     split_value = ident_hash.split(HASH_CHAR)
-    if split_value[0] == '':
-        raise ValueError("Missing values")
 
     try:
+        if split_value[0] == '':
+            raise ValueError("Missing values")
+
         id, version = split_value
```

Comment:
No, this isn't right. This is a ValueError. The value is missing so, it's legitimately an error with the value, not the syntax.